src/main/handlers/helpers/exportUtils.ts

```diff
@@ -25,3 +25,8 @@ export const getExportFilePath: (
 
   return dialogResponse.filePath as string;
 };
+
+export const fracFpsToDec: (fracFps: string) => number = (fracFps) => {
+  const nums = fracFps.split('/').map(Number); // get the numbers (separated by / ), and convert to numbers.
```

Comment:
You can array destructure to give the numbers actual meaning

`const [numerator, denominator] = fracFps.split ...`

Also this requires the input be a certain format or it'll crash, so verify that and/or add a docstring explaining

Finally, the name `fracFpsToDec` is not accurate since this always returns an int (semantically it does, even though JS has no concept of an actual int type). Maybe `fractionToInt` would be more accurate as this isn't necessarily FPS specific either

Reply:
Implemented the changes - `ffprobe` seems to prefer always returning fps in this format, whether average or minimum fps. [Documentation here](https://trac.ffmpeg.org/wiki/FFprobeTips) not sure if we should put this somewhere.